openbb_platform/extensions/derivatives/integration/test_derivatives_api.py

```diff
@@ -139,7 +151,7 @@ def test_derivatives_futures_curve(params, headers):
 
     query_str = get_querystring(params, [])
     url = f"http://0.0.0.0:8000/api/v1/derivatives/futures/curve?{query_str}"
-    result = requests.get(url, headers=headers, timeout=60)
+    result = requests.get(url, headers=headers, timeout=10)
```

Comment:
might be a bad idea to have such a small timeout on an integration test, will probably cause a false positive.